File: code/php/Human/Woman.php
<?php

namespace Human;

require_once __DIR__ . '/Human.php';
require_once __DIR__ . '/../Doer/Runner.php';

use Doer\Runner;

/**
 * Represents a woman
 */
class Woman extends Human implements Runner
{
    /**
     * Begins to run.
     * 
     * @return void
     */
    public function run(): void
    {
        println("{$this->name} is running...");
    }
}

File: code/python/human/woman.py
from .human import Human

class Woman(Human):
    """
    sadsadsadsad
    sad
    sad
    asd
    sad
    sad
    sad
    sa
    dsa
    dsadsadasdsad
    """
    def __init__(self, name, age):
        super().__init__(name, age)
    
    def run(self) -> any:
        println(f"{self.name} is running")

def println(msg: str):
    print(msg)